proto/umee/leverage/v1/events.proto

```diff
@@ -0,0 +1,108 @@
+syntax = "proto3";
+package umee.leverage.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/leverage/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// EventSupply is emitted on Msg/Supply
+message EventSupply {
+  // Liquidity supplier bech32 address.
+  string supplier = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // Liquidity provided to the module.
+  cosmos.base.v1beta1.Coin supplied = 2 [(gogoproto.nullable) = false];
+  // uAsset received by the supplier in exchange for the
+  // provided liquidity.
+  cosmos.base.v1beta1.Coin uasset = 3 [(gogoproto.nullable) = false];
+}
+
+// EventWithdraw is emitted on Msg/Withdraw
+message EventWithdraw {
+  // Liquidity supplier bech32 address.
+  string supplier = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // uAsset sent to the module in exchange for the underlying asset.
+  cosmos.base.v1beta1.Coin uasset = 2 [(gogoproto.nullable) = false];
+  // Liquidity received by the supplier.
+  cosmos.base.v1beta1.Coin asset = 3 [(gogoproto.nullable) = false];
+}
+
+// EventCollaterize is emitted on Msg/Collaterize
+message EventCollaterize {
+  // Borrower bech32 address.
+  string borrower = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // uAsset provided as a collateral.
+  cosmos.base.v1beta1.Coin uasset = 2 [(gogoproto.nullable) = false];
+}
+
+// EventDecollaterize is emitted on Msg/Decollateralize
+message EventDecollaterize {
+  // Borrower bech32 address.
+  string borrower = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // uAsset removed from collateral.
+  cosmos.base.v1beta1.Coin uasset = 2 [(gogoproto.nullable) = false];
```

Comment:
We need to be consistent with Collateral part. Moreover, the request uses `asset`. But probably it makes sense to use `collateral` in both places (EventCollaterize and EventDecollaterize).

Reply:
The reason I'm using `uasset` (changed to `utoken`) here is because in the request we are using `asset` and to make it clear that we can collaterize only uAssets / uTokens.